rlberry/manager/agent_manager.py

```diff
@@ -568,6 +569,31 @@ def set_writer(self, idx, writer_fn, writer_kwargs=None):
         writer_kwargs = writer_kwargs or {}
         self.writers[idx] = (writer_fn, writer_kwargs)
 
+    def generate_profile(self, budget=None, fname="profile.prof"):
```

Comment:
A last small suggestion: if `fname` is None (which could be the default), we can save it to `self.output_dir_ / "profile.prof"`